GameServ/GameHost.cpp

```diff
@@ -154,14 +154,14 @@ void dm_propagate_to(GameHost_Private* host, MOUL::NetMessage* msg,
     DM_UNREFBUF();
 }
 
-void dm_local_sdl_update(GameHost_Private* host, const DS::Blob& blob)
+void dm_local_sdl_update(GameHost_Private* host, DS::Blob blob)
 {
     Auth_NodeInfo sdlNode;
     AuthClient_Private fakeClient;
     sdlNode.m_client = &fakeClient;
     sdlNode.m_internal = true;
     sdlNode.m_node.set_NodeIdx(host->m_sdlIdx);
-    sdlNode.m_node.set_Blob_1(blob);
+    sdlNode.m_node.set_Blob_1(std::move(blob));
```

Comment:
This seems a little non-obvious IMO.... But, as I keep reading downward, I see that `DS::Blob` cannot be copied any more. That's probably for the best in a server.